### ISSUE TITLE
Cannot send mail when connected to ipv6 LTE AP

### ISSUE THREAD

**Issue by kengoldsholl** [k9mail/k-9#953]
When our mail host was moved to RackSpace, I was not able to send mail from K-9 on some of my Android devices while using the LTE connection. If I enabled wi-fi, the mail was sent, otherwise, I would get an error message saying that mail could not be sent. This did not happen before our mail server moved.  I read something about ipv6, and noticed that my T-Mobile connection was using an ipv6 access point. When I changed the protocol used by the access point to ipv4 (I was surprised I was able to do this), the send errors stopped.  

For now, this workaround works, but at some point carriers may not allow the use of ipv4.  I do not know if this is only a problem with RackSpace, but it has happened on every Android device I have used when connected to an ipv6 LTE WAP.

**Comment by cketti** [comment]
Both Android and K-9 Mail should work fine with IPv6. This is more likely a configuration issue than a bug in K-9 Mail, so please use the [mailing list](http://groups.google.com/group/k-9-mail) to ask for help.

Before doing that you might want to make sure that your mail server is reachable via IPv6.

**Comment by kengoldsholl** [comment]
When this happens, the only change I make is the APN protocol in edit access point, and then I am able to send mail. I tried every config option in the send mail server, and all gave the same result. Also, when the only change was enabling wi-fi, and the send mail server was untouched, mail was sent. the incoming mail server is at the same URL, and the devices have no trouble fetching mail when connected to an ipv6 WAP, so it't not a connection issue.  I thought it was a RackSapce problem, but it wasn't happening on all of my phones, and the difference was the APN being used on them - the ones that were able to send mail were connected to a different T-Mo WAP, which only supported ipv4. 

I have only verified that it occurs with T-Mo/ipv6/LTE/RackSpace encrypted mail server/K-9. It could be a T-Mobile or RackSpace problem, but enabling wi-fi (not always available) or using ipv6 (maybe not always available) always fixes it.
